person.rb

```diff
@@ -0,0 +1,37 @@
+require './nameable'
+
+class Person < Nameable
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'unknown', parent_permission: true)
+    super()
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    return true if of_age?
+
+    false
+  end
+
+  def correct_name
+    name
+  end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+    rental.person = self
+  end
```

Comment:
- [ ] Same here, This method supposes to add a new rental not include an already created one to an array, and the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association. Please fix that.